EnglishFilter/sections/foreign.txt

```diff
@@ -3178,6 +3178,8 @@ autopapa.ge##.top-rek
 gamato3.com##iframe[src^="https://syndication.optimizesrv.com/"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/77162
 sport24.gr###someplaceholder
+! https://github.com/AdguardTeam/AdguardFilters/issues/78339
+atnews.gr##.top-footer
```

Comment:
this block mostly contains they own ads

Reply:
Looks like pretty common ads to me, albeit placed on the same domain.

Reply:
formally these elements are their own ads (most of elements) and google-news element is 100% not an ad so we should not hide whole `.top-footer` container